Gemfile.lock

```diff
@@ -330,8 +330,8 @@ GEM
       thor (~> 0.19.1)
     formatador (0.2.5)
     gli (2.15.0)
-    globalid (0.4.0)
```

Comment:
Are these changes to Gemfile.lock intentional? If so, what are they for? If not, could they be reverted?